.pre-commit-config.yaml

```diff
@@ -1,5 +1,11 @@
 repos:
 
+-   repo: https://github.com/pycqa/isort
```

Comment:
@DomenicPuzio Added a subset of our isort checks here.

Reply:
Awesome!